examples/react-next/components/Wallets.tsx

```diff
@@ -6,6 +6,10 @@ import Wallet from './Wallet';
 const Wallets = () => {
   const { wallets } = useWallets();
 
+  if (wallets === null) {
```

Comment:
initializing to null sounds unnecessary. Any reason you could not just check the length of the wallets here?!